packages/zarf-agent/manifests/rolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: zarf-agent-binding
```

Comment:
Why do we set the namespace in the subjects but not here?

Reply:
Good catch, technically doesn't matter since Zarf will set them anyway because the zarf namespace is declared in the manifest block, but I added it for clarity